### ISSUE TITLE
binding different keys to same expression broken

### ISSUE THREAD

**Issue by zhaojiangbin** [abo-abo/hydra#52]
First of all, thanks for the wonderful package. I like it very much.

Just upgraded to version 0.11.0 (melpa version 20150223.1403). Binding different keys to same expression is broken. The following hydra used to work:
```elisp
(bind-key
 "C-c 1"
 (defhydra hydra-foo ()
   "foo"
  ("a" (message "foo"))
  ("b" (message "foo"))
  ("q" nil "quit")))
```
Now pressing "b" gives this error: "Symbol's function definition is void: hydra-foo/lambda-b"

Head a still works just fine.

UPDATE: Replacing the implicit lambda expressions with a defun makes it work again:

```elisp
(defun say-foo ()
  (interactive)
  (message "foo"))

(bind-key
 "C-c 1"
 (defhydra hydra-foo ()
   "foo"
  ("a" say-foo)
  ("b" say-foo)
  ("q" nil "quit")))
```

**Comment by abo-abo** [comment]
Should be fine now. Please check.

**Comment by zhaojiangbin** [comment]
Yes, I can confirm that it has been fixed in the latest MELPA version 20150224.655. Thank you.
